File: 231119/점수대 파악하기/find-out-the-score-range.java
import java.util.*;
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());
        int[] scores = new int[100];
        int[] countArr = new int[11];
        for (int i = 0; i < scores.length; i++) {
            int score = Integer.parseInt(st.nextToken());
            if (score == 0) break;
            countArr[score/10]++; 
        }
        for (int i = countArr.length-1; i > 0; i--) {
            System.out.println(i*10 + " - " + countArr[i]);
        }
    }
}

File: 231119/점수대 파악하기/find-out-the-score-range.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split(' ').map(ele => Number.parseInt(ele));
const zeroValIdx = input.findIndex(ele => ele === 0);
const scores = input.slice(0, zeroValIdx);
const countArr = new Array(11).fill(0);
scores.forEach(score => {
    countArr[Number.parseInt(score / 10)]++;
})
for (const [index, value] of [...countArr.entries()].reverse()) {
    if (index === 0) continue;
    console.log(`${index*10} - ${value}`);
}